### PR TITLE
Initialize tests

### PR DESCRIPTION
**Pull request by chitoku-k**
This PR initializes simple tests that run `git`.

### CODE REVIEW

File: tests/git.zunit [hunk 1]
```diff
@@ -0,0 +1,319 @@
...
+
+    LBUFFER=(git commit '--fixup=**')
+    prefix=--fixup=
+    _fzf_complete_git 'git commit '
+
+    LBUFFER=(git commit '--reedit-message=**')
+    prefix=--reedit-message=
+    _fzf_complete_git 'git commit '
+
+    LBUFFER=(git merge '--reuse-message=**')
```
Comment (hexium310) [inline tests/git.zunit]: It is `commit`, not `merge`.
There are three other places.

Comment (chitoku-k) [reply tests/git.zunit]: 😭

File: tests/git.zunit [hunk 2]
```diff
@@ -0,0 +1,319 @@
...
+    prefix=--reuse-message=
+    _fzf_complete_git 'git commit '
+
+    LBUFFER=(git merge '--squash=**')
+    prefix=--squash=
+    _fzf_complete_git 'git commit '
+}
+
+@test 'Testing git commit -C/--fixup/--reedit-message/--reuse-message/--squash' {
```
[the remark this answers is not in the document]
Comment (chitoku-k) [reply tests/git.zunit]: Good catch 👍

**Comment by hexium310** [comment]
The tests should be run on multiple OS because some commands as `awk` have difference between OS.

**Comment by chitoku-k** [comment]
ToDo:
- Add 'fzf-zsh-completions.plugin.zsh' since no scripts are on the repository root
- Add tests for post-processing

**Pull request merged**